### DISCUSSION TITLE
Bug with Upload field - multiple uploads sort order not saved

### DISCUSSION THREAD

**Discussion by wsdnathan** [orchidsoftware/platform#2264] [General]
Hi @tabuna hoping you can help me figure this one out.

When using the Upload field to batch upload multiple photos/documents, the files all initially process and appear to be in the correct order. However when you save the record and view the multiple attachments again, the order gets jumbled up and the original upload order is not preserved.

Any idea how to get it to save the original upload order as seen before saving the record? Thank you.

**Comment by wsdnathan** [comment]
This doesn't always happen, but sometimes it does. Sorry I know that isn't helpful.

**Reply by tabuna** [reply]
@wsdnathan If you run into this again it would be great if you could give me a reproducible example to define the problem more specifically and solve it.

**Reply by wsdnathan** [reply]
I'll try to figure out what exactly causes this. I think it might have to do with uploading say 100 large images at the same time, each creating multiple thumbnails, and maybe the processing time while handling multiple images concurrently - could this be jumbling it up? Could be entirely unrelated to that just a hunch, because I cannot reliably reproduce the issue yet. Thanks.
